### ISSUE TITLE
Does not show links

### ISSUE THREAD

**Issue by wooyek** [gitaarik/django-admin-relation-links#2]
It does not work at all. What do I do wrong?

```
@admin.register(models.Contact)
class ContactAdmin(AdminChangeLinksMixin, ImportExportMixin, admin.ModelAdmin):
    list_display = ('email', 'phone', 'delivery', 'bill_to', 'contact_type', 'test_enabled')
    list_filter = ('contact_type', 'test_enabled',)
    list_display_links = ('email', 'phone',)
    change_links = ['delivery',]
    list_per_page = 20
```

PACKAGE | NAZWA | WERSJA
-- | -- | --
Django |   | 2.0.3
Python |   | 3.5.2
bootstrapform | Bootstrapform | 3.4
debug_toolbar | Debug Toolbar | 1.9.1
django_assets | Django_Assets | 0.12
django_email_queue | Django email queue | 0.14.0
django_error_views | Django_Error_Views | 0.4.0
django_multiinfo | MultiInfo for Django | 0.1.2
django_opt_out | Messaging Opt-Outs | 0.9.1
django_powerbank | Django_Powerbank | 0.3.5
import_export | Import_Export | 1.0.0

**Comment by belonesox** [comment]
As I understand, this mixin should show links to related objects only on edit page, not on list view.

**Comment by Oxicode** [comment]
+1

**Comment by gitaarik** [comment]
Sorry for the late reply, I just found out I didn't receive notifications for this repository. What exactly does not work? What is `delivery` for type of field?

**Comment by gitaarik** [comment]
I just released 0.1.0, can you try it again?
